lib/client_side_validations/simple_form/form_builder.rb

```diff
@@ -22,6 +22,8 @@ def input(attribute_name, options = {}, &block)
           options.delete(:validate)
         end
 
+        add_field_specific_wrapper_name_to_field_options(attribute_name, options, &block)
```

Comment:
Moving this to the parent gem will not be straightforward. I think I should add the method call to all the overrides

Reply:
Yes, I agree